### PR TITLE
Add debian-iptables and distroless-iptables to ISSUE_TEMPLATE/dep-golang

### PR DESCRIPTION
**Pull request by jingyuanliang**
#### What type of PR is this?

/kind documentation

#### What this PR does / why we need it:

https://github.com/kubernetes/release/issues/2909#issuecomment-1435640173

#### Does this PR introduce a user-facing change?

```release-note
NONE
```

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/dep-golang.md [hunk 1]
```diff
@@ -54,6 +54,20 @@ SIG Release Slack thread:
   - [ ] test image
   - [ ] `.go-version` file
```
Comment (cpanato) [inline .github/ISSUE_TEMPLATE/dep-golang.md:55]: should we add here to update in k/k as well?

Comment (jingyuanliang) [reply .github/ISSUE_TEMPLATE/dep-golang.md:55]: Added an update item there.

**Review by saschagrunert — APPROVED** [review]
Changes LGTM.

/hold
for eyes of @kubernetes/sig-release-leads

**Review by cpanato — APPROVED** [review]
/lgtm

**Review by palnabarun — APPROVED** [review]
/lgtm
/priority important-longterm

**Pull request merged**